molgenis-data/src/test/java/org/molgenis/data/decorator/DynamicRepositoryDecoratorFactoryRegistrarTest.java

```diff
@@ -0,0 +1,35 @@
+package org.molgenis.data.decorator;
+
+import org.springframework.context.ApplicationContext;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.Mockito.*;
+
+public class DynamicRepositoryDecoratorFactoryRegistrarTest
```

Comment:
Please extend from AbstractMockitoTest, and use @Mock for repositoryDecoratorRegistry